input/schemas/paper_source_info.schema.yaml

```diff
@@ -9,7 +9,7 @@ properties:
     type: object
     additionalProperties: false
     properties:
-      livetime: {type: number, unit: hour}
+      livetime: {type: number, unit: hour} # ToDo: create list of livetimes for several observations
```

Comment:
```
# ToDo: create list of livetimes for several observations
```
I don't think we should do that.
We have to "curate" the information, capture the most important one in a uniform schema.

The actual source measurements, spectra, morphology, lightcurves from the sources are our priority now.
Extra information like which telescopes, which livetime, ... is nice to have, but let's keep it simple there and only consider the total for each paper.